lib/room.rb

```diff
@@ -0,0 +1,37 @@
+require 'csv'
+
+module Hotel
+  class Room
+    
+  attr_reader :room_number
+  attr_accessor :price
+
+    def initialize(room_number, price = 200.00)
+      if room_number.class != Integer
+        raise ArgumentError.new("Invalid room number, must be an Integer.")
+      end
+      @room_number = room_number
+
+      if ![Float, Integer].include?(price.class)
```

Comment:
It's more correct to check if `price` is `Numeric` and clearer to use an `unless` here:

```suggestion
      unless price.kind_of? Numeric
```